src/languageSupport/languages/sql/monaco.sql.syntax.ts

```diff
@@ -0,0 +1,23 @@
+import * as pgsql from 'monaco-editor/esm/vs/basic-languages/pgsql/pgsql'
+
+import {MonacoBasicLanguage} from 'src/types'
+
+const LANGID = 'pgsql'
+
+function addSyntax() {
+  monaco.languages.register({
+    id: LANGID,
+  })
+  monaco.languages.setMonarchTokensProvider(
+    LANGID,
+    (pgsql as MonacoBasicLanguage).language
+  )
+  monaco.languages.setLanguageConfiguration(
+    LANGID,
+    (pgsql as MonacoBasicLanguage).conf
+  )
+}
+
+addSyntax()
```

Comment:
I was following the existing pattern, rather than updating. In the existing pattern we loading syntax for a given language whenever the langid for that language is used. This means that when we import the sql `LANGID` into the SqlMonacoEDitor,  the root called `addSyntax()` will load the language features into the monaco-editor.

(Note: this is the existing pattern for all of our languages. I have not re-done this pattern yet.)

This also answers the next question, of why the `FLUXLANGID` is used for the FluxMonacoEditor component. It's so that flux the syntax is loaded into monaco-editor.